registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java

```diff
@@ -1245,7 +1245,11 @@
 
 	RPR_WIA_UNKNOWN_WORKFLOW_ACTION(PlatformConstants.RPR_WORKFLOW_INTERNAL_ACTION + "002", "Unknown Exception"),
 
-	RPR_WIA_SUBWORKFLOW_NOT_FOUND(PlatformConstants.RPR_WORKFLOW_INTERNAL_ACTION + "003", "SubWorkflow not found"),
+	RPR_WIA_ADDITIONALINFOPROCESS_NOT_FOUND(PlatformConstants.RPR_WORKFLOW_INTERNAL_ACTION + "003",
+			"AdditionalInfoProcess not found"),
+
+	RPR_WIA_ADDITIONALINFOPROCESS_CANNOT_REQUEST(PlatformConstants.RPR_WORKFLOW_INTERNAL_ACTION + "004",
+			"AdditionalInfoProcess cannot be  Paused for one more additional info so it is failed"),

```

Comment:
remove the extra space before Paused and make pause with small p

Reply:
fixed